darwin/exporter/formats/nifti.py

```diff
@@ -290,11 +323,15 @@ def unnest_dict_to_list(d: Dict) -> List:
             affine=volume.affine,
         )
         if volume.original_affine is not None:
-            orig_ornt = io_orientation(volume.original_affine)  # Get orientation of current affine
+            orig_ornt = io_orientation(
+                volume.original_affine
+            )  # Get orientation of current affine
             img_ornt = io_orientation(volume.affine)  # Get orientation of RAS affine
-            from_canonical = ornt_transform(img_ornt, orig_ornt)  # Get transform from RAS to current affine
+            from_canonical = ornt_transform(
+                img_ornt, orig_ornt
+            )  # Get transform from RAS to current affine
             img = img.as_reoriented(from_canonical)
-        output_path = Path(output_dir) / f"{image_id}_{volume.series_instance_uid}_{volume.class_name}.nii.gz"
```

Comment:
Sufficiently long filenames resulted in `OSError: [Errno 63] File name too long`, so on the [advice of Jade](https://www.loom.com/share/67a579881e8b418d927c639545779f43?sid=189cb828-b6ec-418f-9bfe-e2096d102f32), remove this UUID (which is V7-generated) from the filenames to reduce the chance of this happening

Reply:
Is this breaking? Won't we be storing filename as part of an annotation that will include the UUID? Then when try to re-read this file elsewhere it won't exist.